customizations/mei-Mensural.xml

```diff
@@ -203,7 +189,63 @@
             <memberOf key="att.duration.quality" mode="add"/>
           </classes>
         </classSpec>
-                
+
+        <macroSpec ident="data.MULTIBREVERESTS.mensural" module="MEI.mensural" type="dt" mode="add">
+          <desc>Logical, that is, written, duration attribute values for multi-breve rests in the mensural repertoire.</desc>
+          <content>
+            <valList type="closed">
+              <valItem ident="2B">
+                <desc>A two-breve rest.</desc>
+              </valItem>
+              <valItem ident="3B">
+                <desc>A three-breve rest.</desc>
+              </valItem>
+            </valList>
+          </content>
+        </macroSpec>
+        <macroSpec ident="data.DURATIONRESTS.mensural" module="MEI.mensural" type="dt" mode="add">
+          <desc>Logical, that is, written, duration attribute values for mensural rests.</desc>
+          <content>
+            <rng:choice>
+              <rng:ref name="data.MULTIBREVERESTS.mensural"/>
+              <rng:ref name="data.DURATION.mensural"/>
+            </rng:choice>
+          </content>
+        </macroSpec>
+
+        <macroSpec ident="data.DURATIONRESTS" module="MEI" type="dt" mode="add">
+          <desc>Logical, that is, written, duration attribute values for rests.</desc>
+          <content>
+            <rng:choice>
+              <rng:ref name="data.DURATION.cmn"/>
+              <rng:ref name="data.DURATIONRESTS.mensural"/>
+            </rng:choice>
+          </content>
+        </macroSpec>
+        <classSpec ident="att.restduration.logical" module="MEI.shared" type="atts" mode="add">
+          <desc>Attributes that express duration of rests in musical terms.</desc>
+          <attList>
+            <attDef ident="dur" usage="opt" mode="change">
+              <desc>Records the duration of a rest using the relative durational values provided by the
+                data.DURATIONRESTS datatype.</desc>
+              <datatype>
+                <rng:ref name="data.DURATIONRESTS"/>
+              </datatype>
+            </attDef>
+          </attList>
+        </classSpec>
+        <classSpec ident="att.rest.log" module="MEI.shared" type="atts" mode="change">
```

Comment:
I don't know if the creation of these two new items **`att.restduration.logical`** and **`data.DURATIONRESTS`** was the most appropriate solution for allowing the "2B" and "3B" values to be used in `<rest>` elements without making them available in the `<note>` elements too. But here is my reasoning behind their use:

The **`att.duration.logical`** class included in **`att.rest.log`** does not include values for encoding "2B" and "3B" rests. Therefore, I substituted it (in the **`att.rest.log`**) by a newly created one: **`att.restduration.logical`**. This new class does include all rests types since it uses the data type **`data.DURATIONRESTS`** which includes both the durations of CMN (**`data.DURATION.cmn`**) and Mensural rests (**`data.DURATIONRESTS.mensural`**).

I didn't want just to add the "2B" and "3B" values to the original **`att.duration.logical`** because that would have made those values available for `<note>` elements too. 

The new **`att.restduration.logical`** and **`data.DURATIONRESTS`** (for rests) are similar in their behavior to the corresponding **`att.duration.logical`** and **`data.DURATION`** for notes. The code of the latter two can be seen here for comparison purposes:
```xml
<macroSpec ident="data.DURATION" module="MEI" type="dt">
  <desc>Logical, that is, written, duration attribute values.</desc>
  <content>
    <rng:choice>
      <rng:ref name="data.DURATION.cmn"/>
      <rng:ref name="data.DURATION.mensural"/>
    </rng:choice>
  </content>
</macroSpec>

<classSpec ident="att.duration.logical" module="MEI.shared" type="atts">
  <desc>Attributes that express duration in musical terms.</desc>
  <attList>
    <attDef ident="dur" usage="opt">
      <desc>Records the duration of a feature using the relative durational values provided by the
        data.DURATION datatype.</desc>
      <datatype>
        <rng:ref name="data.DURATION"/>
      </datatype>
    </attDef>
  </attList>
</classSpec>
```